src/api/routes/v1/amendments.ts

```diff
@@ -43,15 +45,15 @@ interface AmendmentInVoting extends AmendmentsInfo {
 interface AmendmentInVotingMap {
   [key: string]: {
```

Comment:
Is the key here the amendment hash?

Reply:
Yes it is. I used map to parse amendments string "id1,id2,id3" into id1: validator1, validator2, etc. more efficiently